api/DeviceAPI.cs

```diff
@@ -34,18 +35,41 @@ public List<string> enumerateDevices(UInt64 house_id)
 	{
 		//TODO: Verify the input parameters are sufficient
 		//TODO: Implement this function
-		if (house_id == 0)
-	        {
-	            return null;
-	        }
-	        //get device list
-	        var devicelist = new List<string>();
-	        var devices = new List<Device>();
-	        var dlist =
-	            (from device in devices
-	             where device.ID.HouseID == house_id
-	             select device);
-	        return devicelist;
+        if (house_id == 0)
```

Comment:
Ah ok I thought 0 was invalid for some reason
On Apr 23, 2015 10:25 AM, "Jason Ziglar" <notifications@github.com> wrote:

> In api/DeviceAPI.cs
> <https://github.com/ECE4574-5574/Devices/pull/50#discussion_r28966756>:
>
> > @@ -34,18 +35,41 @@ public List<string> enumerateDevices(UInt64 house_id)
> >  	{
> >  		//TODO: Verify the input parameters are sufficient
> >  		//TODO: Implement this function
> > -		if (house_id == 0)
> > -	        {
> > -	            return null;
> > -	        }
> > -	        //get device list
> > -	        var devicelist = new List<string>();
> > -	        var devices = new List<Device>();
> > -	        var dlist =
> > -	            (from device in devices
> > -	             where device.ID.HouseID == house_id
> > -	             select device);
> > -	        return devicelist;
> > +        if (house_id == 0)
>
> ID 0 is a valid House ID.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ECE4574-5574/Devices/pull/50/files#r28966756>.
>